### ISSUE TITLE
Question: How to add pod flipper to custom build configuration #iOS

### ISSUE THREAD

**Issue by IngyuTae** [facebook/flipper#1508]
I have multiple build configurations in my project like,

![41019682-f821752c-695f-11e8-9dd2-cd090d9ea107](https://user-images.githubusercontent.com/24571455/91819767-e6268d80-ec70-11ea-96f2-3462898fd2ce.png)

but it seems `./node_modules/react-native/scripts/react_native_pods.rb` includes `pod filpper*` only for `'Debug'` configuration.

```ruby
  pod 'FlipperKit', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/FlipperKitLayoutPlugin', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/SKIOSNetworkPlugin', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/FlipperKitUserDefaultsPlugin', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/FlipperKitReactPlugin', versions['Flipper'], :configuration => 'Debug'
  # List all transitive dependencies for FlipperKit pods
  # to avoid them being linked in Release builds
  pod 'Flipper', versions['Flipper'], :configuration => 'Debug'
  pod 'Flipper-DoubleConversion', versions['Flipper-DoubleConversion'], :configuration => 'Debug'
  pod 'Flipper-Folly', versions['Flipper-Folly'], :configuration => 'Debug'
  pod 'Flipper-Glog', versions['Flipper-Glog'], :configuration => 'Debug'
  pod 'Flipper-PeerTalk', versions['Flipper-PeerTalk'], :configuration => 'Debug'
  pod 'Flipper-RSocket', versions['Flipper-RSocket'], :configuration => 'Debug'
  pod 'FlipperKit/Core', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/CppBridge', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/FBCxxFollyDynamicConvert', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/FBDefines', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/FKPortForwarding', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/FlipperKitHighlightOverlay', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/FlipperKitLayoutTextSearchable', versions['Flipper'], :configuration => 'Debug'
  pod 'FlipperKit/FlipperKitNetworkPlugin', versions['Flipper'], :configuration => 'Debug'
```

How can I get flipper installed in other configurations? (I don't want to edit `react_native_pods.rb`)

**Comment by stami** [comment]
I just edited the `react_native_pods.rb` file to include my custom configurations:
`:configuration => 'Debug'` -> `:configurations => ['Debug, Development']` or something.

To make it maintainable, use [patch-package](https://github.com/ds300/patch-package) to make a patch file of your changes: `npx patch-package react-native`. That generates a diff file that is applied on `npm install`.

**Comment by IngyuTae** [comment]
solved >= RN 0.63.3 https://github.com/facebook/react-native/commit/dc2df754267df3909631d81c22b9fcab58dfa241
